packages/@roots/bud-dashboard/src/Dashboard/Dashboard.tsx

```diff
@@ -30,11 +30,11 @@ const Dashboard: Dashboard.Component = ({bud}) => {
 
   return (
     <Box flexDirection="column">
-      {appProps.hasErrors && appProps.errors?.length > 0 && (
+      {appProps.errors?.length > 0 && (
         <Static marginBottom={1} items={appProps.errors}>
-          {(err: Dashboard.Compilation.WebpackMessage) => (
+          {(err: Dashboard.Compilation.WebpackMessage, id) => (
             <Module
-              key={err.moduleIdentifier}
+              key={`${id}-webpack-error`}
```

Comment:
the default eslint formatter doesn't actually pass a message object the way native webpack messages work (there is no `moduleIdentifier`, or its empty). So, handling the `key` directly.